plugin.py

```diff
@@ -41,6 +42,15 @@ def on_settings_changed(self, settings: DottedDict) -> None:
             extraPaths.extend(self.find_package_dependency_dirs(py_ver))
             settings.set("python.analysis.extraPaths", extraPaths)
 
+    @classmethod
+    def install_or_update(cls) -> None:
```

Comment:
This is ST4-only since `lsp_utils` doesn't have generic API for this that would work for both ST3 and 4.